snowfall/training/asg_graph.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c)  2020  Xiaomi Corp.       (author: Fangjun Kuang)
+
+from functools import lru_cache
+from typing import Iterable
+from typing import List
+
+import re
+
+import k2
+import torch
+
+from .ctc_graph import build_ctc_topo
+
+
+def filter_disambig_symbols(symbol_table: k2.SymbolTable,
```

Comment:
Mm.  I'd maybe call this `get_phone_symbols` - and of course zero is not going to be a phone symbol but epsilon... should not return zero..

Reply:
This function is also invoked to build ctc_topo, where `0` is a vaid ID, i.e., it is the blank symbol.

Reply:
Will  change it to `get_phone_symbols`. 

Then the user should prepend 0 to the returned result while building ctc topo.